app/quadraticlands/migrations/0001_initial.py

```diff
@@ -40,7 +40,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('created_on', models.DateTimeField(auto_now=True)),
-                ('claim_total', quadraticlands.models.Uint256Field(default=0)),
+                ('claim_total', models.DateTimeField(auto_now=True)),
                 ('distribution', django.contrib.postgres.fields.jsonb.JSONField(default=dict)),
                 ('profile', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='initial_distribution', to='dashboard.Profile')),
             ],
```

Comment:
^ could we undo the changes to the migration file as production already has them

Reply:
Ahhh yeah good call!

When I include the `quadraticlands` model it throws an error and the app won't compile locally. 

I updated it so that it is still a `Uint256Field` but open to suggestions on other ways around that 😄

Reply:
Hmm the issue with that is -> it's not what is running on the server.
Since we'd en up removing this in a follow up PR 
Just to be on the safer side -> maybe we just keep it as original

Reply:
Sounds good and makes sense 🚀 I reverted so there are no changes to this file